packages/design-tokens/tests/__snapshots__/index.spec.ts.snap

```diff
@@ -1,20 +1,20 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Themes light should match the snapshot 1`] = `
-Object {
-  "borderRadius": Object {
+{
+  "borderRadius": {
```

Comment:
...using the built-in matcher just removes these types. I don't think we lose anything here.